src/inspector_agent.cc

```diff
@@ -62,6 +64,21 @@ bool AcceptsConnection(InspectorSocket* socket, const std::string& path) {
                             sizeof(DEVTOOLS_PATH) - 1);
 }
 
+std::string FormatJSON(const std::map<std::string, std::string> object) {
+  std::ostringstream json;
+  json << "[ {\n";
+  bool first = true;
+  for (const auto& name_value : object) {
+    if (!first)
+      json << ",\n";
+    json << "  \"" << name_value.first << "\": \"";
+    json << name_value.second << "\"";
+    first = false;
+  }
+  json << "\n} ]";
+  return json.str();
+}
```

Comment:
v8::JSON::stringify requires V8 context. This code runs on a dedicated (non-V8) thread, can be ran early in the startup (e.g. if the node is started with --debug-brk), needs to run when V8 is suspended on a breakpoint, etc.

I renamed to MapToString. I hope we will not need a "full" JSON serializer :)